pages/research/department/[id].tsx

```diff
@@ -1,5 +1,6 @@
 import * as ScrollArea from "@radix-ui/react-scroll-area";
 import { rollups, scaleOrdinal } from "d3";
+import { useState } from "react";
```

Comment:
Sorry, I didn't notice this in my last review. Adding `state` to a page component, even though it is only used for one very particular thing (the alumni map in this case) on that page increases the risk for unnecessary re-renderings. Hence, for a bad performing page (and all its components). I believe it would be good to  to create an new component which wraps the alumni map (the drop-down element and the map). The state would only be needed within such component.